src/ansi-c/c_qualifiers.h

```diff
@@ -28,7 +29,13 @@ class c_qualifierst
     read(src);
   }
 
-  void clear()
+  c_qualifierst(const c_qualifierst &other) = delete;
```

Comment:
Why?

Reply:
To prevent constructing a `c_qualifiert` from a `java_qualifiert` and consequently losing data.